diff

```diff
@@ -0,0 +1,9 @@
+diff --git a/.gitignore b/.gitignore
+index 762cc89..9f724da 100644
+--- a/.gitignore
++++ b/.gitignore
+@@ -133,3 +133,4 @@ dmypy.json
+ 
+ # pytype static type analyzer
+ .pytype/
++*.swp
```

Comment:
What does this file do? Trying to google it just returns information about git-diff.

Reply:
yes, .gitignore pulled from https://github.com/github/gitignore/blob/master/Python.gitignore
*.swp is vim swap files.

Reply:
I don't mean swap files specifically, I mean the file "diff" in general. I didn't want to need to select the whole file.

Reply:
oh, yeah i don't think that's supposed to be there.